src/shared/ReplicaController.d.ts

```diff
@@ -1,54 +1,47 @@
 import { Replica } from "../index";
 
-type ReplicaClassToken<T extends keyof Replicas> =
-	| symbol
-	| {
-			/**
-			 * @deprecated
-			 * @hidden
-			 */
-			__originalName: T;
-	  };
-
-/**
- * @client
- */
 export interface ReplicaController {
 	/**
 	 * Fired once after the client finishes receiving initial replica data from server.
 	 */
-	InitialDataReceivedSignal: RBXScriptSignal;
+	InitialDataReceivedSignal: RBXScriptSignal<() => void>;
 	/**
 	 * Set to true after the client finishes receiving initial replica data from server.
 	 */
 	InitialDataReceived: boolean;
-
 	/**
-	 * Listens to creation of replicas client-side of a particular class.
-	 *
-	 * This is the preferred method of grabbing references to all replicas clients-side.
+	 * Listens to creation of replicas client\-side of a particular class.
+	 * ```ts
+	 * ReplicaController.ReplicaOfClassCreated("Flower", (replica) => {
+	 *   print(`Flower replica created: ${replica.Identify()}`);
+	 *   print(replica.Class === "Flower") // true
+	 * });
+	 * ```
+	 * This is the preferred method of grabbing references to all replicas clients\-side.
 	 */
 	ReplicaOfClassCreated: <C extends keyof Replicas>(
 		replicaClass: C,
 		listener: (replica: Replica<Replicas[C]["Data"], Replicas[C]["Tags"], Replicas[C]["WriteLib"]>) => void,
 	) => RBXScriptConnection;
 	/**
-	 * Fired every time a replica is created client-side.
+	 * Fired every time a replica is created client\-side.
+	 * ```ts
+	 * ReplicaController.NewReplicaSignal.Connect((replica) => {
+	 *   print(`Replica created: ${replica.Identify()}`);
+	 * })
+	 * ```
 	 */
 	NewReplicaSignal: RBXScriptSignal<(replica: Replica) => void>;
 	/**
-	 * Returns a `Replica` that is loaded client-side with a `Replica.Id` that matches `replicaId`.
+	 * Returns a `Replica` that is loaded client\-side with a `Replica.Id` that matches `id`.
 	 */
-	GetReplicaById: (replicaId: Replica["Id"]) => Replica | undefined;
+	GetReplicaById: (id: Replica["Id"]) => Replica | undefined;
```

Comment:
This should be reverted to keep the parameter names as similar to ReplicaService as possible.